modules/openapi-generator/src/main/resources/rust-axum/apis.mustache

```diff
@@ -72,7 +72,7 @@ pub trait {{classnamePascalCase}} {
     {{#x-consumes-multipart-related}}
         body: axum::body::Body,
     {{/x-consumes-multipart-related}}
-    ) -> Result<{{{operationId}}}Response, String>;
+    ) -> Result<{{{operationId}}}Response, ()>;
```

Comment:
Kindly ping @wing328 to review

This is interface/trait update. The other changed-files are generated samples.

Reply:
thanks for the heads-up

if users need a way to rollback, we can then provide an option or the users can simply use custom templates for the time being.